### DISCUSSION TITLE
[Prettier Plugin] Compatibility with prettier-plugin-jinja-template

### DISCUSSION THREAD

**Discussion by si14** [tailwindlabs/tailwindcss#11297] [Ideas]
Hey there!

Thank you for a great Prettier plugin. Unfortunately, with [the way prettier plugin works](https://github.com/tailwindlabs/prettier-plugin-tailwindcss#compatibility-with-other-prettier-plugins) there has to be an explicit integration with the "parent" plugin that parses the template.

Tailwind is extremely popular with Django community, but so far there is no good way to integrate Prettier with Django templates (you can find quite a few people asking for it [here](https://github.com/prettier/prettier/issues/5581)). However, Django also supports jinja2, which combined with [prettier-plugin-jinja-template](https://github.com/davidodenwald/prettier-plugin-jinja-template) (that seems actively developed) offers a way to have autosorted tailwind classes while using Django on the backend.

Do you folks think you can integrate that plugin in the foreseeable future? It would be a huge help.

**Comment by si14** [comment]
Oh wait, it seems to actually work transparently because that jinja2 plugin delegates to the standard html plugin! I guess it can be resolved, but please let me know if I misunderstand something

**Comment by pySilver** [comment]
I've tried to run it this way:

`prettier sample.jinja2 --write --plugin=prettier-plugin-jinja-template --plugin=prettier-plugin-tailwindcss`

and it does what it is supposed to do for both plugins. However `prettier-plugin-jinja-template` has some troubles with jinja tags that are not supposed to be closed like `{% trans "Foo" %}` So in case of parser failure plugin(s) are not working.

See: https://github.com/davidodenwald/prettier-plugin-jinja-template/issues/16
